rules/DeadCode/Rector/Assign/RemoveUnusedVariableAssignRector.php

```diff
@@ -147,7 +146,7 @@ private function isUsed(Assign $assign, Variable $variable): bool
 
         /** @var FuncCall|MethodCall|New_|NullsafeMethodCall|StaticCall $expr */
         $expr = $assign->expr;
-        if (! $this->isCall($expr)) {
+        if (! $this->sideEffectNodeDetector->detectCallExpr($expr)) {
```

Comment:
Could this also be done in \Rector\DeadCode\Rector\Assign\RemoveDoubleAssignRector::isCall ?

Reply:
Nevermind, didn't see https://github.com/rectorphp/rector-src/pull/679